common/containers/Tabs/SignAndVerifyMessage/components/SignMessage/index.tsx

```diff
@@ -11,6 +11,7 @@ import SignButton from './SignButton';
 import { isWalletFullyUnlocked } from 'selectors/wallet';
 import './index.scss';
 import { TextArea } from 'components/ui';
+import Code from 'components/ui/Code';
```

Comment:
I've seen this a bunch so just dropping a note, but you can just add `Code` to the bulk import of `components/ui` above instead of adding a new import each time. Just add the export to `components/ui/index.tsx`

Reply:
forgot to do that, I've just been having vscode auto import and hadn't noticed